tests/conftest.py

```diff
@@ -10,7 +10,7 @@
 import six
 from setuptools.wheel import Wheel
 
-import pip._internal
+import pip._internal.main
```

Comment:
If we're moving things, it probably makes sense to make this `from ... import`:

```suggestion
from pip._internal.main import main as pip_entry_point
```

Reply:
Done [here](https://github.com/pypa/pip/pull/7061/commits/3e98ee8464ab7e5b63c602bc315c8dea4a63a554#diff-484462fced51d1a06b1d93b4a44dd535R13).